content/docs/manuals/unikernel-operations.md

```diff
@@ -1,12 +1,134 @@
 ---
-title: "Unikernel Operations"
+title: "Unikernel Deployment"
 summary: ""
 draft: false
 weight: 301
 toc: true
 seo:
-  title: "" # custom title (optional)
-  description: "" # custom description (recommended)
+  title: "Unikernel Deployment" # custom title (optional)
+  description: "Deployment of Unikraft Unikernels in Oakestra" # custom description (recommended)
   canonical: "" # custom canonical URL (optional)
   noindex: false # false (default) or true
----
\ No newline at end of file
+---
+
+Oakestra supports the deployment of unikernels built using [Unikraft](http://unikraft.org). 
+
+#### What do you need?
```

Comment:
I think this should be done in the style of the the other requirements callouts such as on the `oak-environment/high-level-setup-overview/` page